plugins/googlecode/build.xml

```diff
@@ -87,7 +87,7 @@
     <target name="compile-tests" depends="compile">
         <mkdir dir="build/classes-tests-ant"/>
 
-        <javac srcdir="src/junit" destdir="build/classes-tests-ant" source="1.5" target="1.5" debug="on" includeantruntime="false">
+        <javac srcdir="src/junit" destdir="build/classes-tests-ant" source="1.8" target="1.8" debug="on" includeantruntime="false">
```

Comment:
this change is harmless (this is just junit tests), but needed. JDK9 refuses to compile with `-source 1.5`, being 1.6 the lowest acceptable:

     [javac] error: Source option 1.5 is no longer supported. Use 1.6 or later.
     [javac] error: Target option 1.5 is no longer supported. Use 1.6 or later.